sdk/translation/azure-ai-translation-text/azure/ai/translation/text/_version.py

```diff
@@ -6,4 +6,4 @@
 # Changes may cause incorrect behavior and will be lost if the code is regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "1.0.0b2"
+VERSION = "1.0.0b1"
```

Comment:
Which version should this be?

Reply:
I think we want to do the beta 2 or full 1.0.0 - I need to confirm with PMs. I will keep beta 2 for now.